### PR TITLE
R&R API Requests + skeleton code, a couple smaller changes too

### PR DESCRIPTION
**Pull request by samboonzaayer**
(sorry if its a lot)

### CODE REVIEW

**Review by boyiq — APPROVED** [review]
Everything looks good to me

File: client/src/components/rr_components/Reviews.jsx [hunk 1]
```diff
@@ -1,9 +1,67 @@
...
+ * text input for summary (60 char), text input for body (50-1000 char, counter for characters),
+ * button to upload photos, text input for nickname, text input for email (60 char), submit btn
+ *
+ * Keyword search: automatically starts filtering the reviews after the user types more than
+ * 2 characters in the search bar
+ *  - The search should work with other filters
+ *  - EC: highlight the text that matches the search result
+ */
+
+const Reviews = ({ id }) => {
```
Comment (boyiq) [inline client/src/components/rr_components/Reviews.jsx]: Isn't the passed in state from App currentItem instead of id?

Comment (samboonzaayer) [reply client/src/components/rr_components/Reviews.jsx]: I'll change that

Comment (boyiq) [reply client/src/components/rr_components/Reviews.jsx]: Awesome, everything else looks good to me!

**Pull request merged**